swagger.json

```diff
@@ -3105,6 +3156,66 @@
                 }
             }
         },
+        "VehicleStats": {
+            "type": "object",
+            "description": "Engine state and digital aux data for a vehicle.",
+            "properties": {
+                "engineState": {
+                    "$ref": "#/definitions/ObjectStatIntSeries"
```

Comment:
Left comment in mini dev spec. Could you add more information on what is the expected response array size? Are object stats created every x seconds, or do we only show object stats for a state change for these boolean values?

Reply:
For `engineState`, we only show object stats for a state change.

Reply:
Is this the same for digital aux data?

Reply:
yes